LightLMS/LLMS
# Generate Django project and run it in docker compose

This PR enables the user to run a django generated project in docker compose locally.

To test: 

  - Pull the code and run run `docker-compose build && docker-compose up`

Expected behavior: The dev server should be spinned up and the default Django page returned.
